support-data-rest/src/test/java/org/uniprot/api/support/data/configure/domain/AdvancedSearchTermIT.java

```diff
@@ -91,6 +106,67 @@ void testFunctionChildren(Integer childNumber, String label) {
         Assertions.assertEquals(FUNCTION_CHILDREN.get(childNumber), label);
     }
 
+
+    @Test
+    void testSiblingGroupFieldHasSiblings() {
+        AdvancedSearchTerm siblingGroupItem = SEARCH_TERMS.stream()
+                .filter(advancedSearchTerm -> advancedSearchTerm.getId().equals("gene_ontology"))
+                .findFirst().orElseThrow(AssertionFailedError::new);
+
+        Assertions.assertNotNull(siblingGroupItem);
+        Assertions.assertEquals("sibling_group", siblingGroupItem.getItemType());
+        Assertions.assertNotNull(siblingGroupItem.getSiblings());
+        Assertions.assertEquals(2, siblingGroupItem.getSiblings().size());
+        Assertions.assertNull(siblingGroupItem.getItems());
+    }
+
+    @Test
+    void testGroupFieldsHasItems() {
+        AdvancedSearchTerm groupItem = SEARCH_TERMS.stream()
+                .filter(advancedSearchTerm -> advancedSearchTerm.getId().equals("subcellular"))
+                .findFirst().orElseThrow(AssertionFailedError::new);
+
+        Assertions.assertNotNull(groupItem);
+        Assertions.assertEquals("group", groupItem.getItemType());
+        Assertions.assertNotNull(groupItem.getItems());
+        Assertions.assertEquals(4, groupItem.getItems().size());
+        Assertions.assertNull(groupItem.getSiblings());
+    }
+
+    @Test
+    void testGeneOntologyEvidences() {
+        AdvancedSearchTerm goEvidence = SEARCH_TERMS.stream()
+                .filter(advancedSearchTerm -> advancedSearchTerm.getId().equals("gene_ontology"))
+                .flatMap(advancedSearchTerm -> advancedSearchTerm.getSiblings().stream())
+                .filter(advancedSearchTerm -> advancedSearchTerm.getId().equals("go_evidence"))
+                .findFirst().orElseThrow(AssertionFailedError::new);
+        Assertions.assertNotNull(goEvidence);
+        Assertions.assertEquals("evidence", goEvidence.getFieldType());
+        Assertions.assertNotNull(goEvidence.getEvidenceGroups());
+        Assertions.assertEquals(GoEvidences.INSTANCE.getEvidences(), goEvidence.getEvidenceGroups());
+    }
+
+    @Test
+    void testAnnotationEvidences() {
+        AdvancedSearchTerm annotationEvidence = SEARCH_TERMS.stream()
+                .filter(advancedSearchTerm -> advancedSearchTerm.getId().equals("function"))
+                .flatMap(advancedSearchTerm -> advancedSearchTerm.getItems().stream())
+                .filter(advancedSearchTerm -> advancedSearchTerm.getId().equals("cofactors"))
+                .flatMap(advancedSearchTerm -> advancedSearchTerm.getItems().stream())
+                .filter(advancedSearchTerm -> advancedSearchTerm.getId().equals("chebi_term"))
+                .flatMap(advancedSearchTerm -> advancedSearchTerm.getSiblings().stream())
+                .filter(advancedSearchTerm -> advancedSearchTerm.getId().equals("ccev_cofactor_chebi"))
+                .findFirst().orElseThrow(AssertionFailedError::new);
+        Assertions.assertNotNull(annotationEvidence);
```

Comment:
It is a long chain. It will be bit tough to understand if you dont see the config file. Can you please put the comment around filter-filterMap